bin/node-template/node/Cargo.toml

```diff
@@ -55,6 +55,9 @@ frame-benchmarking-cli = { version = "4.0.0-dev", path = "../../../utils/frame/b
 # Local Dependencies
 node-template-runtime = { version = "4.0.0-dev", path = "../runtime" }
 
+# CLI-specific dependencies
```

Comment:
There should also be a `try-runtime` feature flag below, and enabling it for the runtime.

Did you test that the binary you build actually does stuff?

Reply:
I've made the changes